doc/HttpLuaModule.wiki

```diff
@@ -6178,7 +6178,7 @@ Timeout for the reading operation is controlled by the [[#lua_socket_read_timeou
 
 <geshi lang="lua">
     sock:settimeouts(1000, 1000, 1000)  -- one second timeout for connect/read/write
-    local data, err = sock:receiveany(10 * 1024 * 1024) -- read any data, at most 10K
+    local data, err = sock:receiveany(10 * 1024 * 1024) -- read any data, at most 10MB
```

Comment:
Imho, `10 * 1024` provides a more reasonable example for this API. I suggest that we update the bytes formula to `10 * 1024` or `10 * 2^10` to read 10KiB.

Reply:
Hi @thibaultcha :). Sure, done. I took the liberty of a squash-rebase at the same time.

Reply:
@lukego Thanks!